File: Primeiro período/AED-1/Java/Trabalho Prático 01 - AED1/AreaCirculo.java
// Autor: Luiz Junio <luisjuniorbr@gmail.com>

import java.util.Scanner;

class AreaCirculo { 
   public static void main (String [] args)
   {  
      Scanner s = new Scanner (System.in);
      System.out.print("Digite o raio do circulo: ");
      double raio = s.nextDouble();
      double area = Math.PI * Math.pow(raio,2);
      System.out.printf("A área do circulo é: %.2f  ", area);
   }
}  

File: Primeiro periodo/AED-1/CSharp/Trabalho Prático 01/AreaCirculo.cs
using System;

public class AreaCirculo {
    public static void Main(){
        Console.Write("Digite o raio do circulo: ");
        double raio = Convert.ToDouble(Console.ReadLine());
        double area = Math.PI * Math.Pow(raio,2);
        Console.WriteLine($"A área do circulo é: {area}");
    }
}